packages/shared/components/atoms/Icon.svelte

```diff
@@ -1,15 +1,15 @@
 <script lang="typescript">
-    import { icons } from './icons'
+    import { ICONS, Icon } from '@lib/auxiliary/icon'
 
-    export let icon = undefined
-    export let width = undefined
-    export let height = undefined
+    export let icon: Icon = undefined
+    export let width: number | number = undefined
+    export let height: number | number = undefined
```

Comment:
Not sue this is correct, should this be number | string? Additionally, with the types we can remove the explicit undefined.

Reply:
updated